### DISCUSSION TITLE
请问有没有办法下载番剧的 pv ？

### DISCUSSION THREAD

**Discussion by vx13** [yutto-dev/bilili#88] [Q&A] [answered]
如[《魔眼收集列车》的特别篇](https://www.bilibili.com/bangumi/play/ep276917)，直接输这个地址会跳转到番剧正传的下载，但剧集里没有特别篇这种归类在 ep-section-module 里的视频。

**Comment by SigureMo** [comment, marked as answer]
emmm 抱歉当前是没有办法的……
